af_data_adoxa_loader/models/calendar_appointment.py

```diff
@@ -0,0 +1,278 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Odoo, Open Source Management Solution, third party addon
+#    Copyright (C) 2004-2019 Vertel AB (<http://vertel.se>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from odoo import models, fields, api, _
+import logging
+_logger = logging.getLogger(__name__)
+
+from odoo.tools import config
+
+import csv
+import os
+import tempfile
+import gc
+
+
+class CalendarOccasion(models.Model):
+    _inherit = "calendar.appointment"
+
+    xmlid_module="__adoxa_import__"
+
+    @api.model
+    def create_occasions(self):     
+        headers_header = ['occasions.csv', 'Notering',  'Trans', 'Odoo']
+        path = os.path.join(config.options.get('data_dir'), 'Adoxa/occasions.csv')
+        path = "/usr/share/odoo-af/af_data_adoxa_loader/data/test_dumps/occasions.csv" #testing purposes only
```

Comment:
path is set twice, why?

Reply:
one is for easy testing, the other for production

Reply:
How do we switch to the production one? 
A new commit that removes the test one?